### PR TITLE
Azure file share for NeuVector

### PR DESCRIPTION
**Pull request by timwebster9**
### JIRA link (if applicable) ###

https://dev.azure.com/hmcts/Platform%20Engineering/_workitems/edit/164

### Change description ###

Add Azure file share for NeuVector persistent storage

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
[AB#164](https://dev.azure.com/hmcts/90472b63-dc8d-4ae6-8819-a32bd211e914/_workitems/edit/164)

### CODE REVIEW

File: scripts/create-azure-file-share.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+RESOURCE_GROUP_NAME=$1
+STORAGE_ACCOUNT_NAME=$2
+
+# Create an azure file share for NeuVector persistent storage
+CONNECTION_STRING=$(az storage account show-connection-string -n ${STORAGE_ACCOUNT_NAME} -g ${RESOURCE_GROUP_NAME} --query 'connectionString' -o tsv)
```
Comment (timja) [inline scripts/create-azure-file-share.sh]: does the connection string need to be stored anywhere? like key vault?

Comment (timwebster9) [reply scripts/create-azure-file-share.sh]: no - it's not really useful for anything but this

Comment (timja) [reply scripts/create-azure-file-share.sh]: i mean for us to be able to get for neuvector?
We won't have direct access to the subscription so won't be able to retrieve it if it isn't stored somewhere

Comment (timwebster9) [reply scripts/create-azure-file-share.sh]: ok will upload to KV - we will need the account key to create the sealed secret for NV

**Pull request merged**